sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -2502,6 +2510,16 @@ private void SetNameFieldsIfNull()
         /// If the destination blob should be sealed.
         /// Only applicable for Append Blobs.
         /// </param>
+        /// <param name="destinationImmutabilityPolicy">
+        /// Optional <see cref="BlobImmutabilityPolicy"/> to set on the blob.
+        /// Note that is parameter is only applicable to a blob within a container that
+        /// has version level worm enabled.
```

Comment:
could we not "use version level worm" anywhere in the codebase?

Reply:
Fixed.